sky/cli.py

```diff
@@ -3783,21 +3791,35 @@ def serve_update(
     memory: Optional[str],
     disk_size: Optional[int],
     disk_tier: Optional[str],
+    mode: str,
     yes: bool,
 ):
     """Update a SkyServe service.
 
+    service_yaml must point to a valid YAML file.
+
     SkyServe will reuse old replicas, if only the service section is changed
     and no file mounts are specified.
     Otherwise, SkyServe will terminate the old replicas and start new replicas.
 
-    service_yaml must point to a valid YAML file.
+    Two update modes are supported:
+    - "rolling": (default) SkyServe will update the service with rolling update,
+        i.e., it will terminate one old replica whenever one new replica is
+        ready. Traffic can be mixed on old and new replicas.
+    - "blue_green": SkyServe will update the service with blue-green update,
+        i.e., it will wait for new replicas to be ready and then terminate old
+        replicas. Traffic will only be switched from old to new replicas after
```

Comment:
```suggestion
        replicas. Traffic will only be served from old to new replicas after
```

Reply:
It might be more clear with `switched` since it means we change from the old to new replicas only after the number of replicas is enough.